capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -111,12 +111,25 @@ class ClubPage extends Component {
       });
   }
 
+  leaveClub = () => {
+    const userID = window.localStorage.getItem('userID');
+    const removeMember = {
+      id: this.props.id,
+      remove_memberIDs: userID,
+    };
+    fetch('/api/clubs', { method: 'put', body: JSON.stringify(removeMember)})
+        .then(this.removeMember(userID))
+        .then(this.props.history.push('/'))
+        .catch(e => console.log(e));
+  }
+
   componentDidMount() {
     this.fetchData();
   }
 
   render() {
-    const isOwner = this.state.owner && this.state.club.ownerID === window.localStorage.getItem('userID');
+    const userID = window.localStorage.getItem('userID');
+    const isOwner = this.state.owner && this.state.club.ownerID === userID;
```

Comment:
I like separating this into 2 lines, it makes it look much cleaner and understandable

Reply:
Thanks!